ci_tools/check_accidental_draft.py

```diff
@@ -0,0 +1,24 @@
+#pylint: disable=unused-import
+""" Script to check if a dev has forgotten to remove the draft status from their PR.
+"""
+import argparse
+from check_stale_state import stale_comment
+from git_evaluation_tools import check_previous_comments, leave_comment
+
+draft_comment = "Your PR is passing all the tests but it is still marked as draft!\n\nDid you forget to remove the draft status? If you are ready to begin the review process then you can go ahead and remove it :rocket:\n\nIf you have got stuck and need a hand finishing your PR don't hesitate to ask the devs on the [Pyccel Discord Server](https://discord.gg/2Q6hwjfFVb)."
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Check if any drafts look like they should be in review')
+    parser.add_argument('pr_number', metavar='pr_number', type=str, nargs='+',
+                            help='File where the markdown output will be printed')
+
+    args = parser.parse_args()
+
+    prs = args.pr_number
+
+    for pr_id in prs:
+        previous_comments, last_comment, last_date = check_previous_comments()
+        if last_comment not in (draft_comment, stale_comment):
+            #TODO: Uncomment before merging and remove pylint disable
+            #leave_comment(pr_id, draft_comment, True)
+            pass
```

Comment:
Should we add something similar for PRs that are not drafts to remind the author to re-request reviews?

Reply:
I think this will be a great help for new developers that are inexperienced to github.

is there something that is done now when the Pr pass all the tests but there are no action performed on it for some time?

Reply:
See points 4 & 5 in the description

Reply:
I'm not sure if that is enough or if something additional needs adding. Possibly it is enough as draft status is more strongly enforced by this PR

Reply:
I this that should be enough to get the attention that the Pr need to be passed to the next step.